usersimcrs/dialogue_management/dialogue_state.py

```diff
@@ -0,0 +1,31 @@
+"""Representation of the dialogue state.
+
+The dialogue state includes the turn count, a list of dialogue acts per turn for
+both the agent and the user, and the belief state. The belief state is a
+dictionary that maps slot names to slot values.
```

Comment:
Nit: as the purpose of the dictionary is not the "mapping" but to store values provided slots, perhaps phrase it slightly differently as "dictionary that holds for each slot (key) the values provided by the user"?

Reply:
Good point, done.